tests/unittests/test_interfaces.py

```diff
@@ -0,0 +1,17 @@
+import torch
+
+
+def test_load_audio(tmpdir):
+    from speechbrain.pretrained.interfaces import Pretrained
+
+    verification = Pretrained.from_hparams(
+        source="speechbrain/spkrec-ecapa-voxceleb",
```

Comment:
It's good to have a unit test on this!
Loading a 100+ MB model from the web is a bit heavy ;-)
Would this snippet from seq2seq integration testing work as well?

https://github.com/speechbrain/speechbrain/blob/d6736adaabf0c6a1021dd1970056f9416bcdcd62/tests/integration/neural_networks/ASR_seq2seq/example_asr_seq2seq_experiment.py#L153

Reply:
```python
from tests.integration.neural_networks.ASR_seq2seq/example_asr_seq2seq_experiment import seq2seqBrain

hparams_file = experiment_dir / "hyperparams.yaml"  # needs to be adjusted
with open(hparams_file) as fin:
    hparams = load_hyperpyyaml(fin)

seq2seq_brain = seq2seqBrain(hparams["modules"], hparams["opt_class"], hparams)
```

it could be lighter and faster to run (especially from scratch which github does with each single PR after a commit)